src/Symfony/Component/DomCrawler/Tests/CrawlerTest.php

```diff
@@ -20,7 +20,10 @@ public function testConstructor()
         $crawler = new Crawler();
         $this->assertCount(0, $crawler, '__construct() returns an empty crawler');
 
-        $crawler = new Crawler(new \DOMNode());
+        $doc = new \DOMDocument();
+        $node = $doc->createElement('test');
```

Comment:
this change should actually be done in older branches too. Passing an orphan DOMNode creates a broken crawler:

- we actually only accept ``DOMElement`` (and ``DOMDocument`` where we get the documentElement in it), not any ``DOMNode`` (you get errors in some cases if you inject such nodes in a crawler)
- we don't accept nodes not attached to a document (you would get an ``Invalid state error`` from the DOM library when accessing ``ownerDocument``, which we already do when filtering the crawler)